test/browser/webgl_offscreen_framebuffer_swap_with_bad_state.c

```diff
@@ -27,6 +27,13 @@ int main() {
 
   EMSCRIPTEN_WEBGL_CONTEXT_HANDLE ctx = emscripten_webgl_create_context("#canvas", &attr);
   emscripten_webgl_make_context_current(ctx);
+#ifdef TEST_DISABLE_VAO
+  EM_ASM({
+    console.log(GL.currentContext.defaultVao);
+    assert(GL.currentContext.defaultVao);
+    GL.currentContext.defaultVao = undefined;
+  });
+#endif
 
 #if !TEST_WEBGL2 && TEST_VAO
```

Comment:
Does anything set TEST_VAO? I can't find any other references to it. I think maybe we want this to be
`#if !TEST_WEBGL2 && !TEST_DISABLE_VAO`?

Reply:
This was fixed in #23064